felix/iptables/table.go

```diff
@@ -1346,6 +1293,88 @@ func (t *Table) applyUpdates() error {
 	return nil
 }
 
+func (t *Table) writeOutBuffer(buf *RestoreInputBuilder, features *environment.Features) error {
```

Comment:
```suggestion
func (t *Table) execIptablesRestore(buf *RestoreInputBuilder) error {
```
Think I'd call this `execIptablesRestore` and could you do `features := t.featureDetector.GetFeatures()` inside the function to avoid passing something that's easy to get?  (I think the features are cached so GetFeatures should be fast.)

Reply:
yeah, right, I has this funtion in util before, not in table :+1: